### DISCUSSION TITLE
Standup (Wednesday, October 19th)

### DISCUSSION THREAD

**Discussion by antonieto** [BBGTC/food-bank-app#2] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Started watching a short React Native course to understand the differences with regular React

**What are your priorities for the next 24 hours?**:
 - Finish it

**Blockers**:
 -

**Shoutouts**:
 - @antonieto for being so proactive

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Helped @BrandonMagana with Fedora issues
 - Reviewed @antonieto PR
 - Research on Django

**What are your priorities for the next 24 hours?**:
 - Start building database models
 - Help with the building of screens

**Blockers**:
 - None

**Shoutouts**:
 - @antonieto  for creating BBGTC org

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on auth implementation

**What are your priorities for the next 24 hours?**:
 - Read more aboute expo-jest (testing for the frontend)
 - Create a PR for login / signup endpoints.
 - Start working on Reto. Interfaces de usuario de la app.

**Blockers**:
 - Bit busy

**Shoutouts**:
 - @antonieto for spending 300 varos in an organization

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Start checking how to deploy Expo app to google's play store
 -  Installed dependencies and tools needed for the project
 
**What are your priorities for the next 24 hours?**:
 -  Set up Fedora 
 -  Work on Figma assignment

**Blockers**:
 - My machine ain't working correctly gg

**Shoutouts**:
 - @CarlosCRG19 for  helping me with Fedora
 - @antonieto for creating the organization and keep working on the project
